Cargo.toml

```diff
@@ -22,4 +22,5 @@ members = [
     "tty",
     "wc",
     "yes",
+    "time",
```

Comment:
Can you add this to all toml files related to the OS that this utility can be compiled? The reason behind this is because no all OS works the same, and sometimes (like fuchsia and haiku) doesn't even has a way to do something. The CI also uses the toml files os respective OS to build and test the crates